18F/identity-idp
# LG-9973: Implement new default Sign In page from A/B test results

## 🎫 Ticket

[LG-9973](https://cm-jira.usa.gov/browse/LG-9973)

## 🛠 Summary of changes

Updates Sign In page to reflect the winner of the recent A/B test (the "tabbed" variant), removing all code associated with the A/B test and unused tests.

Easier to review with whitespace changes hidden: https://github.com/18F/identity-idp/pull/8542/files?w=1

## 📜 Testing Plan

1. Go to http://localhost:3000
2. Verify that you can still sign in and create an account

## 👀 Screenshots

Before|After
---|---
![image](https://github.com/18F/identity-idp/assets/1779930/29c3f252-fe92-4f35-8b97-6d5a589a8471)|![image](https://github.com/18F/identity-idp/assets/1779930/d7286d78-d047-43ec-93b1-0716e06487cc)
